themes/vocabulary_theme/templates/blog-series.html

```diff
@@ -9,10 +9,10 @@
     <div class="header">
       <div class="container">
         <h1>All posts in series "{{ this.name }}"</h1>
-        <a class="button small margin-top-bigger" href="{{ this.parent|url }}">See all series</a>
+        <a class="button small margin-top-bigger" href="{{ this.parent|url }}" aria-label="See all series">{{ this.parent.name }}</a>
       </div>
     </div>
-    <div class="series-list container">
+    <div class="series-list container" role="list" aria-label="List of posts in the series">
```

Comment:
it would be more semantically sound to utilize a `ul` than to try to turn a `div` into the role of a list via `aria`